### PR TITLE
Camexplore

### PR DESCRIPTION
**Pull request by Walid-Rovo**
Fix minor issue: from default behaviour to launch camera only, to FT sensor and camera.

### CODE REVIEW

File: ur_description/urdf/inc/ur_common.xacro [hunk 1]
```diff
@@ -29,10 +29,10 @@
   -->
   <xacro:macro name="read_model_data" params="joint_limits_parameters_file kinematics_parameters_file physical_parameters_file visual_parameters_file">
     <!-- Read .yaml files from disk, load content into properties -->
-    <xacro:property name="config_joint_limit_parameters" value="${load_yaml(joint_limits_parameters_file)}"/>
-    <xacro:property name="config_kinematics_parameters" value="${load_yaml(kinematics_parameters_file)}"/>
-    <xacro:property name="config_physical_parameters" value="${load_yaml(physical_parameters_file)}"/>
-    <xacro:property name="config_visual_parameters" value="${load_yaml(visual_parameters_file)}"/>
+    <xacro:property name="config_joint_limit_parameters" value="${xacro.load_yaml(joint_limits_parameters_file)}"/>
```
Comment (cangorur) [inline ur_description/urdf/inc/ur_common.xacro]: Those are already there in the merged branch: https://github.com/micropsi-industries/universal_robot/pull/2

**Review by cangorur — CHANGES_REQUESTED** [review]
Please be careful with what you are merging in the end. A good practice would be to go over the commits one by one and see what they are changing in the end. Currently, the changes in `ur_common.xacro` and the file `ur_ft_camera.xacro` are already merged in the main branch.

File: ur_description/urdf/inc/ur_common.xacro (hunk 1 above)
Comment (Walid-Rovo) [reply ur_description/urdf/inc/ur_common.xacro]: Cleaned up the PR to include only one commit with just the needed change.

**Review by cangorur — APPROVED** [review]
lgtm

**Pull request merged**